### PR TITLE
bugfix(grid) grid refactor - fixing bug at production - vuex store is…

### PR DESCRIPTION
**Pull request by derpdead**
… undefined. Removed Vuex from Grid

### CODE REVIEW

File: modules/products/router.config.js [hunk 1]
```diff
@@ -179,18 +193,31 @@ export const routing = [
                 title: 'Products',
                 icon: 'Document',
             },
-            breadcrumbs: [
-                {
-                    title: 'Products',
-                    icon: 'Document',
-                },
-            ],
+            breadcrumbs: [],
```
Comment (bleto) [inline modules/products/router.config.js]: to remove, we don't need empty array

File: service/grid/getData.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { COLUMNS_IDS } from '~/defaults/grid/cookies';
+
+export const getGridData = ($axios, $cookies, path, params) => $axios
+    .$get(path, { params })
+    .then(({
+        collection,
+        columns,
+        info: { count, filtered },
+    }) => {
+        const sortedColumns = params.columns ? getSortedColumnsByIDs([...columns], params.columns, 'id') : columns;
```
Comment (bleto) [inline service/grid/getData.js]: Do you need to use a spread operator on the `columns` variable ? You copy array ?

File: core/components/Grid/AdvancedFilters/GridAdvancedFilter.vue [hunk 3]
```diff
@@ -143,53 +139,43 @@ export default {
...
+        },
         selectFooterComponent() {
-            switch (this.data.type) {
+            switch (this.filter.type) {
             case TYPES.SELECT:
                 return () => import('~/core/components/Inputs/Select/Contents/Footers/SelectContentFooter');
             case TYPES.MULTI_SELECT:
             case TYPES.DATE:
-            case TYPES.PRICE:
+            case 'RANGE':
```
Comment (bleto) [inline core/components/Grid/AdvancedFilters/GridAdvancedFilter.vue]: why `RANGE` variable is not in the defaults ?

File: core/components/Grid/Grid.vue [hunk 4]
```diff
@@ -243,63 +283,57 @@ export default {
...
+            const advancedFiltersValues = {};
+
+            for (let i = 0; i < length; i += 1) {
+                if (!this.advancedFilters[i].isGhost
+                    && Object.keys(this.advancedFilters[i].value).length > 1) {
+                    advancedFiltersValues[this.advancedFilters[i].id] = true;
+                }
+            }
+
+            return advancedFiltersValues;
```
[the remark this answers is not in the document]
Comment (derpdead) [reply core/components/Grid/Grid.vue]: It's conditionally assigning - it's faster xD

**Pull request merged**